debian/control

```diff
@@ -15,21 +15,38 @@ Standards-Version: 4.5.1
 
 Package: pimodbus-master
 Architecture: any
+Section: oldlibs
+Depends:
+  revpi-modbus-client,
+  ${misc:Depends}
+Description: transitional package
+  This is a transitional package. It can safely be removed.
+
+Package: pimodbus-slave
+Architecture: any
+Depends:
+  revpi-modbus-server,
+  ${misc:Depends}
```

Comment:
Shouldn't the new packages have Replaces and Breaks?
https://wiki.debian.org/RenamingPackages
```
 Package: newname
 Replaces: oldname (<< 2.0-1~)
 Breaks: oldname (<< 2.0-1~)
 …
```

Reply:
Sorry, I've changed this already. As I said: pull request was too early.